app/controllers/api/databases.php

```diff
@@ -596,10 +597,15 @@ function updateAttribute(
 
         $filterQueries = Query::groupByType($queries)['filters'];
 
-        $response->dynamic(new Document([
-            'databases' => $dbForProject->find('databases', $queries),
-            'total' => $dbForProject->count('databases', $filterQueries, APP_LIMIT_COUNT),
-        ]), Response::MODEL_DATABASE_LIST);
+        try {
+            $response->dynamic(new Document([
+                'databases' => $dbForProject->find('databases', $queries),
+                'total' => $dbForProject->count('databases', $filterQueries, APP_LIMIT_COUNT),
+            ]), Response::MODEL_DATABASE_LIST);
+        } catch (OrderException $e) {
+            $message = "The order attribute '{$e->getAttribute()}' had a null value. Cursor pagination requires all documents order attribute values are non-null.";
+            throw new Exception(Exception::DATABASE_QUERY_ORDER_NULL, $message);
+        }
```

Comment:
Let's move the response->dynamic call after the try/catch so only the find/count calls are inside, for all changes